src/category_theory/limits/preserves/basic.lean

```diff
@@ -256,6 +256,23 @@ Note that we do not assume a priori that `D` actually has any colimits.
 class reflects_colimits (F : C ⥤ D) : Type (max u₁ u₂ (v+1)) :=
 (reflects_colimits_of_shape : Π {J : Type v} {𝒥 : small_category J}, by exactI reflects_colimits_of_shape J F)
 
+/--
+A convenience function for `reflects_limit`, which takes the functor as an explicit argument to
```

Comment:
Wasn't this change also in one of the other PRs you made?

Reply:
The `preserves_limit` version was, not this version (I'm pretty sure)